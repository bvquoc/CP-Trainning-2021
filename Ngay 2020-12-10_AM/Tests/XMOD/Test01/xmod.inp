266 87
4349 5488 964 4962 6966 984 294 3435 2469 294 4900 2336 6152 4086 1507 4768 251 603 4945 6814 2181 3202 3994 1287 1007 365 3043 3990 1606 5769 167 4917 6800 1453 6608 5094 3102 6548 4002 351 1251 5202 5817 6495 2215 5107 719 5559 6663 910 6637 3427 1992 2685 2865 417 2308 1521 3328 5870 2792 1128 1192 5412 2920 4992 3637 6004 2669 5977 6391 5219 2479 5308 5886 3929 4993 4598 877 2857 1289 3739 5745 4265 4561 371 6025 3888 2004 1898 3645 6432 959 2600 5594 1224 2473 3656 3708 2301 1794 2014 3973 392 6933 3082 6072 6378 6823 1675 1718 4202 4950 6451 461 6800 4924 4164 2442 4207 6692 1385 1812 4382 6728 5821 2289 2518 5158 92 2877 3951 2354 799 4756 3474 5802 3957 5087 5205 6734 6151 6467 6628 6891 1955 3952 955 4565 3128 3472 2688 1571 5006 1796 6422 418 4474 6505 65 3848 4514 3640 3637 5792 3429 4351 6476 3603 4814 3090 540 6766 2645 2630 2338 4874 2061 4980 4983 712 574 120 6861 3330 2001 2409 107 3285 3278 5311 574 5924 5649 2492 6317 6036 3151 6897 5254 4580 5028 6715 6474 58 542 2168 3160 2225 4802 4396 5507 2957 3287 3206 3320 6904 4155 688 3334 288 6084 2674 2544 2777 4295 3493 472 1617 4959 1344 5399 4694 827 3289 860 5508 4288 3527 5415 2965 3897 1858 2588 6958 5964 4682 4809 3262 4109 6795 1567 5522 4101 5552 4622 2417 3771 4702 5021 3690 4111 6653 3632 5884 3645 
5615 60 4280 877 6889 3803 1618 2881 330 2022 6688 1582 2591 3117 1851 457 1056 5194 163 516 3041 6501 2406 13 6631 681 1895 220 72 6480 6855 5343 878 2242 4924 5617 2254 5064 4274 1245 3148 6165 5312 5965 6711 3990 139 6801 3997 4607 3196 4343 3921 4866 3114 349 4672 3062 2865 4386 5855 5476 5997 4612 1078 3347 6283 2212 2262 367 1138 2194 1114 5255 4344 1680 5795 3513 4967 554 1076 1387 2285 6533 5090 181 1669 