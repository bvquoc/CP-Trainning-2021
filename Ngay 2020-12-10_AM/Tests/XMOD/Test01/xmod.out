47 5615
2 60
46 4280
7 877
45 6889
33 3803
52 1618
39 2881
40 330
50 2022
18 6688
16 1582
39 2591
43 3117
53 1851
51 457
12 1056
32 5194
47 163
52 516
25 3041
5 6501
28 2406
13 13
19 6631
43 681
39 1895
46 220
14 72
42 6480
11 6855
7 5343
8 878
38 2242
52 4924
49 5617
50 2254
18 5064
40 4274
27 1245
16 3148
17 6165
34 5312
49 5965
41 6711
46 3990
23 139
15 6801
53 3997
25 4607
6 3196
51 4343
35 3921
52 4866
40 3114
1 349
32 4672
46 3062
23 2865
36 4386
55 5855
24 5476
23 5997
30 4612
34 1078
41 3347
19 6283
8 2212
0 2262
19 367
36 1138
48 2194
12 1114
35 5255
52 4344
56 1680
53 5795
33 3513
37 4967
32 554
32 1076
53 1387
23 2285
37 6533
44 5090
7 181
45 1669
